docker/btcd/start-btcd.sh

```diff
@@ -44,8 +44,12 @@ RPCPASS=$(set_default "$RPCPASS" "devpass")
 DEBUG=$(set_default "$DEBUG" "info")
 NETWORK=$(set_default "$NETWORK" "simnet")
 
-PARAMS=$(echo \
-    "--$NETWORK" \
+PARAMS=""
+if [ "$NETWORK" != "mainnet" ]; then
```

Comment:
this seems like a separate issue and maybe should be a different PR?  `start-btcd` is also in need of updating https://github.com/lightningnetwork/lnd/blob/master/docker/btcd/start-btcd.sh#L48

Reply:
https://github.com/lightningnetwork/lnd/issues/2337